### ISSUE TITLE
Change Repository Type

### ISSUE THREAD

**Issue by slysnivy** [shoc71/Keep-On-Going#452]
Our repository is currently a "public template" when it should be just "public", because we're not making a template for a game, we're making a game.

**Comment by shoc71** [comment]
Agreed! Let's reflect those changes before the game "comes out" but we can do it whenever we need to. Also, we could make this current repository private and make a public version of this as well. 

@slysnivy  Thoughts on this idea?
